lxd/storage/backend_lxd.go

```diff
@@ -6230,6 +6242,20 @@ func (b *lxdBackend) BackupCustomVolume(projectName string, volName string, tarW
 		return err
 	}
 
+	contentDBType, err := VolumeContentTypeNameToContentType(volume.ContentType)
+	if err != nil {
+		return err
+	}
+
+	contentType, err := VolumeDBContentTypeToContentType(contentDBType)
+	if err != nil {
+		return err
+	}
+
+	if contentType != "" && contentType != drivers.ContentTypeFS && contentType != drivers.ContentTypeBlock {
```

Comment:
I think we can drop the `contentType != ""` bit here, as if `contentType` is empty then something weird is going on right, and we shouldn't continue?

Reply:
Right, `contentType` will never be empty.